app/views/support_interface/performance/unavailable_choices.html.erb

```diff
@@ -1,18 +1,29 @@
 <% content_for :title, 'Applications with unavailable choices' %>
-<% content_for :before_content, govuk_back_link_to(support_interface_performance_path) %>
 
-<h2 class='govuk-heading-m'>Applications to courses that no longer are available on Apply</h2>
+<% content_for :before_content do %>
+  <%= render BreadcrumbComponent.new(items: [
+    {
+      text: 'Performance',
+      path: support_interface_performance_path
+    },
+    {
+      text: 'Applications with unavailable choices'
+    }
+  ]) %>
+<% end %>
```

Comment:
Do we need this breadcrumb? I figured the first crumb duplicates the navigation, and the second one the page title.

Reply:
It’s both repetitive and yet consistent! I’d actually like to split this particular page up into three as it’s quite long.